test/functionalTest/cases/2015_notification_templates/notification_templates_cache_refresh.test

```diff
@@ -185,6 +185,7 @@ Date: REGEX(.*)
                 "url": "http://127.0.0.1:9997/${NOTIFY_URL}"
             },
             "lastNotification": "REGEX(.*)",
+            "lastSuccess": "REGEX(.*)",
```

Comment:
This requires futher investigation... I understand we have fixed lastSuccess, etc. for the cache enabled case. In this context, this addition makes sense.

However, in the cache disabled case lastSuccess should have appear from the very beginning. Note that jenkins tests both "branches" (cache enabled and disabled) for all test. Why jenkins has not report a fail here before?